### ISSUE TITLE
fatal error: 'bux/LR1.h' file not found

### ISSUE THREAD

**Issue by yurivict** [buck-yeh/parsergen#6]
Build fails:
```
In file included from /disk-samsung/freebsd-ports/devel/parsergen/work/parsergen-1.7.0-9-gbda59af/ParserGen/Parser.cpp:5:
In file included from /disk-samsung/freebsd-ports/devel/parsergen/work/parsergen-1.7.0-9-gbda59af/ParserGen/ParseFile.h:3:
/disk-samsung/freebsd-ports/devel/parsergen/work/parsergen-1.7.0-9-gbda59af/ParserGen/Parser.h:6:10: fatal error: 'bux/LR1.h' file not found
#include <bux/LR1.h>
         ^~~~~~~~~~~
```

**Comment by buck-yeh** [comment]
If you follow the steps of [from github in any of Linux distros](https://github.com/buck-yeh/parsergen#from-github-in-any-of-linux-distros), they are inaccurate right now, please add 2 steps befoe `cmake .` and see if it works:

1. Checkout [`bux`](https://github.com/buck-yeh/bux) manually to `/disk-samsung/freebsd-ports/devel/parsergen/work/bux`
2. Checkout [`CBrackets`](https://github.com/buck-yeh/CBrackets) manually to `/disk-samsung/freebsd-ports/devel/parsergen/work/CBrackets`

I am fixing the repo. Hope soon we will see tag 1.7.1

**Comment by buck-yeh** [comment]
That is the thing I have just learned recently. Definitely will consider it!

**Comment by buck-yeh** [comment]
This issue is closed with the same root cause with #8
